R/as-data-frame.R

```diff
@@ -1,34 +1,64 @@
-CrunchDataFrame <- function (dataset) {
+CrunchDataFrame <- function (dataset, order) {
```

Comment:
1) I'd suggest a different name for "order"; I didn't realize it was for permuting/selecting row indices until I read below; also, it's the name of a function.

2) It appears below that you assume `NULL` is the default, but none is specified here.